src/test/javascript/spec/component/iris/settings/iris-global-settings-update.component.spec.ts

```
@@ -68,6 +73,7 @@
         fixture.detectChanges();
         const irisSettings = mockSettings();
         irisSettings.id = undefined;
+        irisSettings.irisProactivitySettings!.eventSettings.forEach((eventSetting) => (eventSetting.id = undefined));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)